packages/i18next/src/lib/InternationalizationHandler.ts

```diff
@@ -141,6 +141,11 @@ export class InternationalizationHandler {
 			this.languages.set(item, i18next.getFixedT(item));
 		}
 		this.languagesLoaded = true;
+
+		const formatters = this.options.formatters ?? [];
+		for (const { name, format } of formatters) {
+			i18next.services.formatter!.add(name, format);
```

Comment:
Why do you need the `!`?

Reply:
formatter doesn't know if it's initialized, so we have to convince TS that we know we're using this function after we initalized i18n.